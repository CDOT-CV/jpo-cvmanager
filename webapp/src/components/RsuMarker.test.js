import React from "react";
import { render } from "@testing-library/react";
import RsuMarker from "./RsuMarker";

it("should take a snapshot", () => {
  const { asFragment } = render(<RsuMarker />);

  expect(asFragment(<RsuMarker />)).toMatchSnapshot();
});
