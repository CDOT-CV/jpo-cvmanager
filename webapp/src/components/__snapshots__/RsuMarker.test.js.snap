// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot online offline 1`] = `
<div>
  <div
    style="padding: 5px; display: inline-block; border-radius: 50%; width: 5px; height: 5px; background-color: rgb(233, 79, 55);"
  />
</div>
`;

exports[`snapshot online online 1`] = `
<div>
  <div
    style="padding: 5px; display: inline-block; border-radius: 50%; width: 5px; height: 5px; background-color: rgb(161, 211, 99);"
  />
</div>
`;

exports[`snapshot online other 1`] = `
<div>
  <div
    style="padding: 5px; display: inline-block; border-radius: 50%; width: 5px; height: 5px; background-color: rgb(176, 176, 176);"
  />
</div>
`;

exports[`snapshot online unstable 1`] = `
<div>
  <div
    style="padding: 5px; display: inline-block; border-radius: 50%; width: 5px; height: 5px; background-color: rgb(209, 167, 17);"
  />
</div>
`;

exports[`snapshot other 1`] = `
<div>
  <div
    style="padding: 5px; display: inline-block; border-radius: 50%; width: 5px; height: 5px; background-color: rgb(176, 176, 176);"
  />
</div>
`;

exports[`snapshot scms 0 1`] = `
<div>
  <div
    style="padding: 5px; display: inline-block; border-radius: 50%; width: 5px; height: 5px; background-color: rgb(176, 176, 176);"
  />
</div>
`;

exports[`snapshot scms 1 1`] = `
<div>
  <div
    style="padding: 5px; display: inline-block; border-radius: 50%; width: 5px; height: 5px; background-color: rgb(161, 211, 99);"
  />
</div>
`;

exports[`snapshot scms other 1`] = `
<div>
  <div
    style="padding: 5px; display: inline-block; border-radius: 50%; width: 5px; height: 5px; background-color: rgb(176, 176, 176);"
  />
</div>
`;
